src/MasterMix/pipettingLossTableColumn.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { Tooltip } from '../Tooltip';
+
+import { AbsolutePipettingLoss, PipettingLossByFactor } from './types';
+
+export function pipettingLossTableColumn(props: {
+  count: number;
+  pipettingLoss: PipettingLossByFactor | AbsolutePipettingLoss;
+}) {
+  const pipettingLossTitle = (): string => {
+    switch (props.pipettingLoss.type) {
+      case 'absolute':
+        return `${props.pipettingLoss.count}x`;
+      case 'factor':
+        return `${props.pipettingLoss.factor * 100}%`;
+    }
+  };
```

Comment:
On every render, this code:
- redefines this function
- calls it

Use `useMemo` to calculate it depending on changes to `props.pipettingLoss`.